hal/targets/hal/TARGET_NORDIC/TARGET_NRF5/gpio_api.c

```diff
@@ -116,7 +109,7 @@ static void gpio_apply_config(uint8_t pin)
                 if ((m_gpio_irq_enabled & (1 << pin))
                     && (m_gpio_cfg[pin].irq_rise || m_gpio_cfg[pin].irq_fall))
                 {
-                    nrf_drv_gpiote_in_event_enable(pin, false);
+                    nrf_drv_gpiote_in_event_enable(pin, true);
```

Comment:
I've just left it as it was originally. But I think it can make sense, in a hypothetical case, when `gpio_apply_config()` is called after `gpio_irq_disable()`. Thus, I'd prefer to not change it, just in case.